<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewznabSettings</class>
 <widget class="QWidget" name="NewznabSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Newznab</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Newznab</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Cantarell'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt;&quot;&gt;Newznab is protocol which allows client applications to search for Usenet content and view RSS content feeds. Typically Newznab services/servers require registration. Below you can import a list of currently known servers. &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt;&quot;&gt;Visit&lt;/span&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:8pt;&quot;&gt; &lt;/span&gt;&lt;a href=&quot;http://www.newznab.com&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; text-decoration: underline; color:#0000ff;&quot;&gt;www.newznab.com&lt;/span&gt;&lt;/a&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:8pt;&quot;&gt; for more information.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tabAccounts">
         <attribute name="title">
          <string>Acounts</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QListWidget" name="listServers"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnImport">
              <property name="toolTip">
               <string>Import a list of Newznab servers from the Internet</string>
              </property>
              <property name="text">
               <string>Import</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnAdd">
              <property name="text">
               <string>Add ...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnDel">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Edit ...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabRSS">
         <attribute name="title">
          <string>Categories</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QGridLayout" name="gridLayout_12">
            <item row="3" column="1">
             <widget class="QGroupBox" name="groupBox_13">
              <property name="title">
               <string>TV</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_15">
               <item>
                <widget class="QCheckBox" name="chkTvInt">
                 <property name="text">
                  <string>International</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="chkTvSD">
                 <property name="text">
                  <string>SD</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="chkTvHD">
                 <property name="text">
                  <string>HD</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QGroupBox" name="groupBox_12">
              <property name="title">
               <string>Movies</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_14">
               <item>
                <widget class="QCheckBox" name="chkMoviesInt">
                 <property name="text">
                  <string>International</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="chkMoviesSD">
                 <property name="text">
                  <string>SD</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="chkMoviesHD">
                 <property name="text">
                  <string>HD</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="chkMoviesWMV">
                 <property name="text">
                  <string>WMV</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QGroupBox" name="groupBox_17">
              <property name="title">
               <string>Games</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_19">
               <item>
                <widget class="QCheckBox" name="chkConsoleNintendo">
                 <property name="text">
                  <string>Nintendo</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="chkConsolePlaystation">
                 <property name="text">
                  <string>Playstation</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="chkConsoleXbox">
                 <property name="text">
                  <string>XBox</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QGroupBox" name="groupBox_16">
              <property name="title">
               <string>Adult</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_18">
               <item>
                <widget class="QCheckBox" name="chkXXXDVD">
                 <property name="text">
                  <string>DVD</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="chkXXXSD">
                 <property name="text">
                  <string>SD</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="chkXXXHD">
                 <property name="text">
                  <string>HD</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QGroupBox" name="groupBox_15">
              <property name="title">
               <string>Music</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_17">
               <item>
                <widget class="QCheckBox" name="chkMusicMP3">
                 <property name="text">
                  <string>MP3</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="chkMusicLosless">
                 <property name="text">
                  <string>Losless</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="chkMusicVideo">
                 <property name="text">
                  <string>Video</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QGroupBox" name="groupBox_14">
              <property name="title">
               <string>Apps</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_21">
               <item>
                <widget class="QCheckBox" name="chkComputerPC">
                 <property name="text">
                  <string>PC</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="chkComputerISO">
                 <property name="text">
                  <string>ISO</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="chkComputerMac">
                 <property name="text">
                  <string>Mac</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="chkComputerAndroid">
                 <property name="text">
                  <string>Android</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
